client/task-list/list.js

```diff
@@ -54,15 +54,29 @@ export class TaskList extends Component {
 		this.possiblyTrackCompletedTasks();
 	}
 
+	getUngroupedTasks() {
+		const { tasks: groupedTasks } = this.props;
+		return Object.values( groupedTasks ).flat();
+	}
+
+	getSpecificTasks() {
```

Comment:
This seems strange to me.. why are we passing more tasks to a `<TaskList/>` than it's supposed to render?